src/Http/Headers/src/EntityTagHeaderValue.cs

```diff
@@ -53,7 +53,7 @@ public EntityTagHeaderValue(StringSegment tag, bool isWeak)
                 (length != tag.Length))
             {
                 // Note that we don't allow 'W/' prefixes for weak ETags in the 'tag' parameter. If the user wants to
-                // add a weak ETag, he can set 'isWeak' to true.
+                // add a weak ETag, 'isWeak' can be set to true.
```

Comment:
```suggestion
                // add a weak ETag, they can set 'isWeak' to true.
```

This being a code comment, the "style" doesn't matter as much (being inclusive is still good though). But since I can just throw a GitHub suggestion in to update it, I did ;P.

Reply:
This is what I had originally before the suggestion to passive. Thanks for the suggestion.